apps/src/sites/studio/pages/course_offerings/edit.js

```diff
@@ -12,8 +12,19 @@ function showCourseOfferingEditor() {
     scriptData.dataset.courseOfferingEditor
   );
 
+  const scriptSelfPacedData = document.querySelector(
```

Comment:
Good job following the existing pattern here. For the future, we have this lovely getScriptData util that does a lot of this work for us:

https://github.com/code-dot-org/code-dot-org/blob/151f54cea749731032780d403b95be12603a1559/apps/src/util/getScriptData.js#L18-L27